docs/index.md

```diff
@@ -80,6 +82,20 @@ const gentSVG = cityMap(geoData)
 svg`${gentSVG}`
 ```
 
+## Rankings
+```js
+const years = getYears().map(String);
+const categories = getCategories();
+const year = view(Inputs.select(years, {value: "2018", label: "Selecteer jaar:"}));
+const category = view(Inputs.select(categories, {value: "Graffiti", label: "Selecteer categorie:"}));
```

Comment:
Same as above, maybe add a default value to see the results for all crimes at once?